Packs/DeveloperTools/ReleaseNotes/1_3_21.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Create-Mock-Feed-Relationships
+
+- Updated *Indicator Expiration Method*
+- Updated the Docker image to: *demisto/feed-performance-test:1.0.103518*.
```

Comment:
docker wasnt upgraded in the yml

Reply:
Not by me but it did change from 1.3.20 by Daud in July. I guess RN just wasn't updated when it did.